src/elfpy/wallet.py

```diff
@@ -12,6 +12,41 @@
     from typing import Any
 
 
+@dataclass
+class Long:
+    """An open long position.
+
+    Arguments
+    ---------
+    balance : float
+        The amount of longs that are open.
+    """
+
+    balance: float
+
+    def __str__(self):
+        return f"Long(balance: {self.balance})"
+
+
+@dataclass
+class Short:
+    """An open short position.
+
+    Arguments
+    ---------
+    balance : float
+        The amount of shorts that are open.
```

Comment:
same here, weird sentence.  I assume these are in bonds.  margin below is in base?